### ISSUE TITLE
escape_string on a quoted parameter

### ISSUE THREAD

**Issue by garaud** [semirook/snaql#2]
Hi,

I've a query with a `WHERE` keyword and a jinja2 quoted variable that I want to update when I call the "query" method.

```sql
{% sql 'artists', note='get artists' %}
SELECT name,age,instrument
FROM artists
WHERE creation_date = '{{current_date}}'
{% sql %}
```

Python-side, I've got:

```python
q = snaql_factory.load_queries('artists.sql')
q.artists(current_date='2015-10-07')
```
returns
`"SELECT name,age,instrument FROM artist WHERE current_date = \\'2015-10-07\\'"` because there is a call to the `escape_string` function. As is, this SQL query raises an exception such as `OperationalError: unrecognized token : "\"`.

Maybe there is a other/better to handle this. Tell me if you know.
I think the same thing can occur if you just have a `WHERE type = 'email'` and you call the `query` method with a keyword parameter (since this implies an `escape_string` call).

Cheers,
Damien G.

**Comment by semirook** [comment]
What DB is used? I need to test it myself.

**Comment by garaud** [comment]
The previous exception is raised by SQLite but I've got the same issue with SQLServer.

**Comment by semirook** [comment]
Thanks a lot for this request. I've fixed that in 0.3.3 with 34babeb75a0eef9574c8e9162d569a0452cabab1

You can set quotes in the template or use guards.string to escape string and set quotes:

```sql
{% sql 'artists', note='get artists' %}
    SELECT name,age,instrument
    FROM artists
    WHERE creation_date = {{ current_date|guards.string }}
{% sql %}
```

will produce correct SQL string, in your case:

```sql
"SELECT name,age,instrument FROM artist WHERE current_date = '2015-10-07'"
```

**Comment by garaud** [comment]
Pretty well, thanks!!  Works well for me. Moreover, it's documented and tested, perfect!
